components/style/traversal.rs

```diff
@@ -229,15 +229,6 @@ fn ensure_node_styled_internal<'a, N, C>(node: N,
 {
     use properties::longhands::display::computed_value as display;
 
-    // Ensure we have style data available. This must be done externally because
-    // there's no way to initialize the style data from the style system
-    // (because in Servo it's coupled with the layout data too).
-    //
-    // Ideally we'd have an initialize_data() or something similar but just for
-    // style data.
-    debug_assert!(node.borrow_data().is_some(),
```

Comment:
Hmm... I guess there isn't an easy way to restore the equivalent of this assertion (which is better than "tried to unwrap None").

Can you keep the comment somewhere at least, saying that this expects the node data to be initialized?